pkg/logql/parser_test.go

```diff
@@ -534,19 +534,143 @@ func TestParse(t *testing.T) {
 		{
 			in: `{foo="bar"} "foo"`,
 			err: ParseError{
-				msg:  "syntax error: unexpected STRING, expecting != or !~ or |~ or |=",
```

Comment:
Why is this gone ?

Reply:
After the the parser changes, we can no longer assume that a filter expressions is the only possible following set of tokens after the log selector expression. The grammar now supports things like `Expr or Expr`, for instance.